script/playbackstrategy/msestrategy.js

```diff
@@ -67,10 +65,15 @@ define('bigscreenplayer/playbackstrategy/msestrategy',
       }
 
       function onTimeUpdate () {
+        failoverTime = mediaElement.currentTime - timeCorrection;
         publishTimeUpdate();
       }
 
       function onError (event) {
+        if (event.error && event.error.data) {
```

Comment:
Can you elaborate on this? I've seen the commit message 'remove error data we do not need for now' but more context would be useful.

Reply:
I'm assuming this is a hygiene thing that was spotted, for the extra data dashjs sends through on its error event (vs the one from the media element), and not directly related to CDN failover? Is this correct @jlks?

Reply:
Yep, that is correct. 

There is a deep clone of the error event object that happens as a part of the plugin code path that can't handle some of the nested objects that are returned from that data object (Request/Response etc).